### PR TITLE
Added a FillValue for u, v to allow profile submission to IOOS

### PR DESCRIPTION
**Pull request by hvdosser**
As part of the `if` loop where the profile-averaged glider speed _u_ and _v_ are defined, I added an `else` statement to set _u_ and _v_ to a `FillValue` defined in the metadata yaml file, if ocean current speed isn't available (as is usually the case for realtime data).

### CODE REVIEW

File: pyglider/ncprocess.py [hunk 1]
```diff
@@ -67,6 +67,12 @@ def extract_timeseries_profiles(inname, outdir, deploymentyaml):
 
                     dss['v'] = dss.water_velocity_northward.mean()
                     dss['v'].attrs = profile_meta['v']
+                elif ('water_velocity_eastward' not in dss.keys()) and ('_FillValue' in profile_meta['u']):
```
Comment (jklymak) [inline pyglider/ncprocess.py:70]: What is `'_FillValue'` usually?  I'm not familiar with how that gets set up...

Comment (jklymak) [reply pyglider/ncprocess.py:70]: If this is _not_ meta data do we want to fill with NaN anyways, or just not have the variable at all?

Comment (hvdosser) [reply pyglider/ncprocess.py:70]: IOOS suggests -999 for a _FillValue, and it is specified in the variable-specific metadata for u and for v. 

I think if we don't specify it in the metadata, we probably don't want the variable at all. That allows people who don't care about e.g. submitting to IOOS to run the code without unused variables taking up space in their netcdf files.

Comment (jklymak) [reply pyglider/ncprocess.py:70]: Thats fine.  Seems a little strange for FillValue to be a variable-specific metadata, but maybe different sensors need different fill values.

**Pull request merged**